Makefile

```diff
@@ -22,14 +22,7 @@ endif
 SRC_DIR=./cmd
 DST_DIR=./build
 
-CLI_SRC=$(SRC_DIR)/gladius-cli
```

Comment:
@speedaemon Are these fine to remove?

Reply:
I didn't see them in use and there's no clear reason why they'd be needed in the near future so I just removed them.